ddtrace/tracer/span_event.go

```
@@ -229,9 +235,13 @@ func intSliceValue[T constraints.Integer](values []T) *spanEventAttribute {
 func floatSliceValue[T constraints.Float](values []T) *spanEventAttribute {
 	arrayVal := make([]*spanEventArrayAttributeValue, 0, len(values))
 	for _, v := range values {
+		f := float64(v)
+		if math.IsNaN(f) || math.IsInf(f, 0) {
+			continue // Skip non-finite values
```

Does this mean that some event attributes will get silently dropped?